src/components/Reviews.jsx

```diff
@@ -5,40 +5,71 @@ import { Link, useParams, useSearchParams } from 'react-router-dom';
 const Reviews = () => {
 const [isLoading, setIsLoading] = useState(true);
 const [reviews, setReviews] = useState([]);
-const [category, setCategory] = useSearchParams();
-const categoryType = category.get('category')
+const [searchParams, setSearchParams] = useSearchParams();
+const sortbyQuery = searchParams.get('sort_by');
+const orderQuery = searchParams.get('order');
+
 
 useEffect(() => {
     setIsLoading(true);
-    getReviews(categoryType).then(({ reviews }) => {
+    getReviews(sortbyQuery, orderQuery).then(({ reviews }) => {
         setReviews(reviews);
         setIsLoading(false);
     });
-}, [categoryType]);
+}, [sortbyQuery, orderQuery]);
 
 if (isLoading) {
     return <p className="Loading">Loading...</p>;
 };
 
+const setSearch = (sort_by, order) => {
+    const newParams = new URLSearchParams(searchParams);
+    if (sort_by) {newParams.set('sort_by', sort_by)};
+    if(order) {newParams.set('order', order)};
+    setSearchParams(newParams);
+  };
+
+
 return (
+<div>
+<section>
+    <select className="sortby-dropdown" value={sortbyQuery} onChange={(e) => {
+        setSearch(e.target.value, null)
+        }}>
+    <option disabled="disabled" selected={true} hidden={true}>Sort By</option>        
+    <option value="title">Title</option>
+    <option value="owner">Reviewer</option>
+    <option value="category">Category</option>
+    <option value="designer">Designer</option>
+    <option value="created_at">Date</option>
+    </select>
+    <select className="orderby-dropdown" value={orderQuery} onChange={(e) => {
+        setSearch(null, e.target.value)
+    }}>
+    <option disabled="disabled" selected={true} hidden={true}>Order</option> 
+    <option value="asc">Ascending</option>
+    <option value="desc">Descending</option>
+    </select>
+</section>
 <section class="cards-wrapper">
 {reviews.map((review) => { 
-return  <div class="card-grid-space" key={review.review_id}>
+return  <div className="card-grid-space" key={review.review_id}>
         <Link to={`/reviews/${review.review_id}`} className="card-link">
-        <a class="card" style={{"--bg-img": `url(${review.review_img_url})`}}>
+        <a className="card" style={{"--bg-img": `url(${review.review_img_url})`}}>
```

Comment:
This shouldn't be an `a` tag. You're using `Link`s to navigate to avoid hard refreshes. Swap it for a div if it's just for the background image (which is a lovely effect btw 😁 )